src/assets/styles/base.scss

```diff
@@ -269,13 +269,13 @@ ul.list {
 
 // General icon styling
 .header-icon {
-  size: 25px !important;
+  font-size: 25px !important;
```

Comment:
Do we have font size variables that can be used?

Reply:
Will look 👀

Reply:
We don't but i'll add them to the theme, apply them here :)

Reply:
Updated, lemme know if that is what you meant

Reply:
Perfect.

I know it might seem silly to name a variable after its value like this, but if we use those only for font sizes then it's easy to do a global search to see where they're used (compared with hard values). And it will be easy to perhaps prune our font sizes down to 4 instead of 6 (or more in other UIs), when UI team goes through and asks for cleanup.

Reply:
Aside: Here's an example of something we should clean up some time:

https://github.com/bcgov/business-create-ui/blob/main/src/assets/styles/theme.scss#L67